ln-simln-jamming/src/main.rs

```diff
@@ -267,58 +266,26 @@ async fn main() -> Result<(), BoxError> {
         })
         .collect();
 
-    let attacker_actions_shutdown = shutdown.clone();
-    let attack_custom_actions = Arc::clone(&attack);
-    let custom_actions_listener = listener.clone();
-    // Channel to signal when the attack is ready to start calling the `simulation_completed`
-    // check. Sender will be passed to `run_custom_actions` and signal from there that the check
-    // for the attack shutdown criteria can be started.
-    let (tx, mut rx) = oneshot::channel();
+    let main_attack = Arc::clone(&attack);
```

Comment:
Unnecessary clone, we can allow `attack` to be moved by the task spawn because it's not needed again.